set-client/pom.xml

```diff
@@ -36,15 +36,30 @@
 			<artifactId>jettison</artifactId>
 			<version>${version.jettison}</version>
 		</dependency>
-		
+
+		<!-- https://mvnrepository.com/artifact/jakarta.ws.rs/jakarta.ws.rs-api -->
+		<dependency>
+			<groupId>jakarta.ws.rs</groupId>
+			<artifactId>jakarta.ws.rs-api</artifactId>
+			<version>4.0.0</version>
```

Comment:
this dependency is not needed, only the UriBuilder class is used in BaseApiConnection. The use of UriBuilder needs to be replaced by the use of org.apache.hc.core5.net.URIBuilder from the httclient

Reply:
removed , used httpclient 5 uri builder